### PR TITLE
🍒 [SE-0412] changelog note about strict concurrency for global variables

### PR DESCRIPTION
**Pull request by sophiapoirier**

### CODE REVIEW

**Comment by sophiapoirier** [comment]
@swift-ci please test

**Comment by sophiapoirier** [comment]
@swift-ci please test

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,6 +5,16 @@
 
 ## Swift 5.10
 
+* [SE-0412][]:
+
+  Under strict concurrency checking, every global or static variable must be either isolated to a global actor or be both immutable and of `Sendable` type.
```
Comment (hborla) [inline CHANGELOG.md:10]: Can you show an example of some code that previously could exhibit data races at runtime that is now diagnosed under the proposal?

**Comment by sophiapoirier** [comment]
@swift-ci please test

**Comment by sophiapoirier** [comment]
@swift-ci please test

**Comment by sophiapoirier** [comment]
> The changelog item is added under **Swift 5.10** here, but under **Swift 5.11** on the main branch.

Ah good point, I modeled the same as was done with SE-0411 without much thinking about it. I have now on main also moved SE-0411 under 5.10 (@hborla I hope you agree that is the correct change).

> In both pull requests, a reference is missing (at the end of the changelog):
> 
> ```
> [SE-0412]: <https://github.com/apple/swift-evolution/blob/main/proposals/0412-strict-concurrency-for-global-variables.md>
> ```

Thanks and fixed now.

**Comment by hborla** [comment]
> I have now on main also moved [SE-0411](https://apple.github.io/swift-evolution/#?proposal=SE-0411) under 5.10 (@hborla I hope you agree that is the correct change).

Yes I agree, thank you! At the time when I landed SE-0411 on `main`, it was not yet on 5.10, so the `main` release note became incorrect after I cherry picked the change.

**Review by hborla — APPROVED** [review]

**Comment by sophiapoirier** [comment]
@swift-ci please test macOS

**Pull request merged**